solutions/findloop.js

```diff
@@ -0,0 +1,11 @@
+const findLoop = (obj, current=obj)=>{
+  if(obj.next === current){
```

Comment:
Lint errors are when the left side of your terminal has red marks. Next time fix those lint errors and also do proper indentation.
if (obj.next == current) {